### PR TITLE
Simplify SetThreadName

### PR DESCRIPTION
**Pull request by wendigo**
# Airlift contribution check list

 - [x] Git commit messages follow https://cbea.ms/git-commit/.
 - [x] All automated tests are passing.
 - [x] Pull request was categorized using one of the existing labels.

### CODE REVIEW

**Comment by martint** [comment]
What's the motivation for this?

**Comment by wendigo** [comment]
In all callsites we just concatenate string with a number or two numbers so
it seems an overkill for such usecase

**Review by electrum — APPROVED** [review]

File: concurrent/src/main/java/io/airlift/concurrent/SetThreadName.java [hunk 1]
```diff
@@ -25,8 +25,10 @@ public class SetThreadName
     public SetThreadName(String threadNamePrefix)
     {
         requireNonNull(threadNamePrefix, "threadNamePrefix is null");
-        originalThreadName = Thread.currentThread().getName();
-        Thread.currentThread().setName(threadNamePrefix + "-" + Thread.currentThread().threadId());
+        Thread currentThread = Thread.currentThread();
+        long threadId = currentThread.threadId();
```
Comment (electrum) [inline concurrent/src/main/java/io/airlift/concurrent/SetThreadName.java:29]: This could be inlined

Comment (wendigo) [reply concurrent/src/main/java/io/airlift/concurrent/SetThreadName.java]: Done

**Pull request merged**